public/src/components/trading_terminal/charts/Charts.tsx

```diff
@@ -195,7 +195,7 @@ export class Charts extends React.Component<ChartsProps, ChartsState> {
 			$("#volume-chart-container").addClass("active");
 		}
 
-		this.getOldStockHistory(this.props.stockId);
+		await this.getOldStockHistory(this.props.stockId);
```

Comment:
I tested it, it worked fine. Though in line 204 we are retrieving without await, i tried changing intervals, but is working fine

Reply:
Yes the bug has never appeared on changing intervals, so I think the lack of an `await` in line 204 shouldn't be an issue